protractor.config.js

```diff
@@ -34,7 +34,7 @@ exports.config = {
   useAllAngular2AppRoots: true,
 
   // Base URL for application server
-  baseUrl: 'http://localhost:8080',
+  baseUrl: 'http://localhost:3000',
```

Comment:
We do not need to start a lite-server with browser-sync, which will be started with npm start and run on port 3000. It is enough to have a simple http server you can start with  npm run serve which will run on port 8080.